### ISSUE TITLE
[Bug] Inline title color not consistent with h1 color

### ISSUE THREAD

**Issue by lihua-cat** [AnubisNekhet/AnuPpuccin#264]
**Describe the bug**
Inline title color is not consistent with h1 color in Style Settings

**To Reproduce**
1. Toggle `Enable Custom Heading Colors` in Style Settings
2. Toggle `Show Inline Title` in obsidian option/apperance
3. See error

**Expected behavior**
Same font color of inline title and h1

**Screenshots**
![img](https://github.com/AnubisNekhet/AnuPpuccin/assets/42488653/aeb053a9-8ce0-4760-88a9-a0217739bdf2)

**Comment by ProudBenzene** [comment]
In-line titles are not the same as H1 headings in Obsidian. If you want to change the style of the inline titles, you need to make a separate change to the CSS associated with the inline titles, which has nothing to do with the style of the H1 header. They are independent.

If you wish to modify the styling of in-line headings, you can refer to this article and create a new css snippet in the .obsidian/snippets folder to do so: [Obsidian 样式：笔记页内标题居中]( https://pkmer.cn/show/20230619155040 )

**Comment by lihua-cat** [comment]
You're right. They are different things. I just thought the inline title styles inherit from the H1 headings. Since it`s not true, feel free to close this issue.
